developer-docs-site/docs/releases/devnet-release.md

```diff
@@ -13,6 +13,10 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
 
 [Aptos Framework Release v1.2.0](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.2.0)
 
+## Aptos SDK
+
+Aptos SDK Release v1.7.2
+
 ## Aptos CLI
 
 |Release | Git Tag | Commit Hash|
```

Comment:
Let's add 1.0.8!  I just released it :)

Reply:
Can I get the values from you or somewhere else?

Reply:
All done.